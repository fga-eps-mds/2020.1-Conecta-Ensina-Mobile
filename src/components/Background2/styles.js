import styled from 'styled-components';

export const BlueContainer = styled.View`
  background-color: ${(props) => props.theme.colors.fundoAzul};
  flex: 0.7;
  justify-content: flex-end;
  justify-content: center;
  align-items: center;
  width: 100%;
`;

export const WhiteContainer = styled.View`
  background-color: ${(props) => props.theme.colors.branco};
  justify-content: center;
  align-items: center;
  flex: 1;
  border-top-left-radius: 20px;
  border-top-right-radius: 20px;
  width: 100%;
`;

export const Container = styled.View`
  background-color: ${(props) => props.theme.colors.fundoAzul}
  justify-content: center;
  align-items: center;
  flex: 1;
`;
