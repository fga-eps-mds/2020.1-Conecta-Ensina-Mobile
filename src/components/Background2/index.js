import React from 'react';

import {BlueContainer, WhiteContainer} from './styles';

export default function Background2({children}) {
  return (
    <BlueContainer>
      <WhiteContainer>{children}</WhiteContainer>
    </BlueContainer>
  );
}
