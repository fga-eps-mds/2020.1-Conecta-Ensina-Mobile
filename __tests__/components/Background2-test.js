import React from 'react';
import 'react-native';
import renderer from 'react-test-renderer';

import Background2 from '../../src/components/Background2';

it('renders correctly', () => {
  const tree = renderer.create(<Background2 />).toJSON();
  expect(tree).toMatchSnapshot();
});
