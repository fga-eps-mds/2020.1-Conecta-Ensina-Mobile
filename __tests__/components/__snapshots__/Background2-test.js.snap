// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "alignItems": "center",
        "backgroundColor": "#184E79",
        "flexBasis": 0,
        "flexGrow": 1,
        "flexShrink": 1,
        "justifyContent": "center",
      },
    ]
  }
  testID="background"
>
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#184E79",
          "flexBasis": 0,
          "flexGrow": 0.7,
          "flexShrink": 1,
          "justifyContent": "center",
          "width": "100%",
        },
      ]
    }
  />
  <View
    style={
      Array [
        Object {
          "alignItems": "center",
          "backgroundColor": "#ffffff",
          "borderTopLeftRadius": 20,
          "borderTopRightRadius": 20,
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "center",
          "width": "100%",
        },
      ]
    }
  />
</View>
`;
